src/js/utils.js

```diff
@@ -22,12 +22,12 @@ export const chartExportOptions = {
         }
         window.open(url, '_blank');
       },
-      text: 'Show Query'
+      text: 'Show query'
     }
   },
   buttons: {
     contextButton: {
-      menuItems: ['showQuery', 'downloadPNG']
+      menuItems: ['showQuery']
```

Comment:
What's this change about?

Reply:
The "Download PNG image" button is extremely slow and it doesn't even work
<img width="1185" alt="image" src="https://github.com/HTTPArchive/httparchive.org/assets/1120896/4e60d006-1460-4a9e-9228-418b042cf7d9">